### ISSUE TITLE
Error using DisplayUndistortionLabRIggerMouseStim

### ISSUE THREAD

**Issue by adamimos** [Psychtoolbox-3/Psychtoolbox-3#351]
Hello,

I am trying to create a spherical distortion map for visual stimulus (for a mouse), and am using the DisplayUndistortionLabRiggerMouseStim code from Tubigen. I have it working on one computer but for some reason when using it on another I get the following error. Does anyone have any idea what the issue is?

Thanks,
Adam

```
Error using CreateDisplayWarp>BVLGeneratetextcoord (line 712)
Not enough input arguments.

Error in CreateDisplayWarp>CSVComputeWarpMesh (line 775)
[xyzcalibpos, xytexcoords] = BVLGeneratetextcoord(yLoomSize, xLoomSize, textureCoords,
vertexCoords, showCalibOutput);

Error in CreateDisplayWarp (line 437)
       [xyzcalibpos, xytexcoords] = CSVComputeWarpMesh(winWidth, winHeight, calib.scal,
       showCalibOutput);

Error in PsychImaging>PostConfiguration (line 3358)
               [warpstruct, filterMode] = CreateDisplayWarp(win, calibfilename,
               showCalibOutput, reqs{row, 5:10});

Error in PsychImaging (line 2054)
   PostConfiguration(reqs, win, clearcolor, slavewin);

Error in moving_ball_phase0 (line 79)
[window, windowRect] = PsychImaging('OpenWindow', screenNumber, gray);
```

**Comment by dcnieho** [comment]
You'll have to do some debugging, to step through the functions (
http://de.mathworks.com/help/matlab/matlab_prog/debugging-process-and-features.html)
and see where one of the needed input arguments is not generated probably.
Or just check the call reateDisplayWarp>CSVComputeWarpMesh (line 775) and
see if it provides all input arguments the BVLGeneratetextcoord function
needs

**Comment by kleinerm** [comment]
There was a bug in that function, which was fixed in the PTB 3.0.12 beta release from 25th July 2015. Your failing computer has an outdated PTB, therefore the failure.
